bleach/__init__.py

```diff
@@ -347,6 +347,13 @@ def link_repl(match):
         if url.startswith('('):
             _wrapping = strip_wrapping_parentheses(url)
             url, open_brackets, close_brackets = _wrapping
+        elif url.endswith('/)'):
+            # This is a very common corner case where the URL is listed
+            # last within a longer text within parentheses.
+            # Example: (check my site at http://foo.bar/)
+            # Do not consider the closing ) as part of the URL.
+            url, open_brackets, close_brackets = url[:-1], 0, 1
```

Comment:
I see what happened. You wrote up the issue and then tossed something more in this PR. It's a lot easier to work through pull requests when they don't have a pile of changes in them.